cider/tests/utils/DuckDbQueryRunner.cpp

```diff
@@ -388,6 +390,164 @@ void addColumnDataToCiderBatch<CiderByteArray>(
   }
 }
 
+template <typename T>
+void addColumnDataToScalarBatch(std::unique_ptr<duckdb::DataChunk>& dataChunk,
+                                int i,
+                                int row_num,
+                                CiderBatch* child) {
+  CHECK(child->resizeBatch(row_num, true));
+  auto data_buffer = child->asMutable<ScalarBatch<T>>()->getMutableRawData();
+  auto null_buffer = child->getMutableNulls();
+  int64_t null_count = 0;
+
+  for (int j = 0; j < row_num; j++) {
+    T value = std::numeric_limits<T>::min();  // init with Cider null value
+    if (dataChunk->GetValue(i, j).IsNull()) {
+      CiderBitUtils::clearBitAt(null_buffer, j);
+      ++null_count;
+    } else {
+      value = dataChunk->GetValue(i, j).GetValue<T>();
+      CiderBitUtils::setBitAt(null_buffer, j);
+    }
+    std::memcpy(data_buffer + j, &value, sizeof(T));
+  }
+  child->setNullCount(null_count);
+}
+
+template <>
+void addColumnDataToScalarBatch<CiderDateType>(
```

Comment:
I think you can remove this since `CiderDateType` will be replaced by Arrow date representation. You can add this method in later PRs. Same as L443 one.

Reply:
OK. So both `CiderDateType` and `CiderByteArray` should be removed right?